tests/Metalama.Framework.Tests.Integration/Tests/Aspects/Introductions/Types/IntoNamespace.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Metalama.Framework.Aspects;
+using Metalama.Framework.Code;
+
+namespace Metalama.Framework.Tests.Integration.Tests.Aspects.Introductions.Types.IntoNamespace
+{
+
+    public class IntroductionAttribute : TypeAspect
+    {
+        public override void BuildAspect(IAspectBuilder<INamedType> builder)
+        {
+            builder.Advice.IntroduceClass(builder.Target.ContainingNamespace, "TestType", TypeKind.Class);
+        }
```

Comment:
There should be a test demonstrating that you are able to introduce two types in a new namespace and that this results in two different files.